### PR TITLE
2.0.0 RTW install links

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL 1](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?branch=pr-en-us-3946#prerequisites)
[Review URL 2](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?branch=pr-en-us-3946&tabs=aspnet20)

Sorry about the trailing space noise.

### CODE REVIEW

File: aspnetcore/mvc/razor-pages/index.md [hunk 1]
```diff
@@ -13,21 +13,19 @@ uid: mvc/razor-pages/index
...
 If you're looking for a tutorial that uses the Model-View-Controller approach, see [Getting started with ASP.NET Core MVC](xref:tutorials/first-mvc-app/start-mvc).
 
-[Download or view sample code](https://github.com/Rick-Anderson/razor-page-intro).
-
 <a name="prerequisites"></a>
 
 ## ASP.NET Core 2.0 preview prerequisites
 
-ASP.NET Core 2.0 preview  is included in .NET Core 2.0 preview. [Download .NET Core 2.0 Preview 2](https://www.microsoft.com/net/core/preview) 
+ASP.NET Core 2.0 preview is included in .NET Core 2.0 preview. [Download .NET Core 2.0 Preview 2](https://www.microsoft.com/net/core/preview)
```
Comment (scottaddie) [inline aspnetcore/mvc/razor-pages/index.md]: Remove the references to preview

File: aspnetcore/mvc/razor-pages/index.md [hunk 2]
```diff
@@ -13,21 +13,19 @@ uid: mvc/razor-pages/index
...
-Razor Pages requires ASP.NET Core 2.0.0 or later. Tooling support for Razor Pages ships in Visual Studio 2017 Update 3 or later.
+Razor Pages requires [.NET Core 2.0](https://dot.net/core) or later. Tooling support for Razor Pages ships in [Visual Studio 2017](https://www.visualstudio.com/downloads/) Update 3 or later.
 
 If you're looking for a tutorial that uses the Model-View-Controller approach, see [Getting started with ASP.NET Core MVC](xref:tutorials/first-mvc-app/start-mvc).
 
-[Download or view sample code](https://github.com/Rick-Anderson/razor-page-intro).
-
 <a name="prerequisites"></a>
 
 ## ASP.NET Core 2.0 preview prerequisites
```
Comment (scottaddie) [inline aspnetcore/mvc/razor-pages/index.md]: Remove "preview" from heading

File: aspnetcore/includes/install2.0.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Install the following:
+
+* [.NET Core 2.0.0 SDK](https://dot.net/core) or later.
+* [Visual Studio](https://www.visualstudio.com/vs/) 15.3 or later.Add the seed initializer to the end
```
Comment (scottaddie) [inline aspnetcore/includes/install2.0.md]: Add 1 space before "Add the seed"

File: aspnetcore/mvc/razor-pages/index.md [hunk 4]
```diff
@@ -13,21 +13,19 @@ uid: mvc/razor-pages/index
 ---
 # Introduction to Razor Pages in ASP.NET Core
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Ryan Nowak](https://github.com/rynowak) 
+By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Ryan Nowak](https://github.com/rynowak)
 
 Razor Pages is a new feature of ASP.NET Core MVC that makes coding page-focused scenarios easier and more productive.
 
-Razor Pages requires ASP.NET Core 2.0.0 or later. Tooling support for Razor Pages ships in Visual Studio 2017 Update 3 or later.
+Razor Pages requires [.NET Core 2.0](https://dot.net/core) or later. Tooling support for Razor Pages ships in [Visual Studio 2017](https://www.visualstudio.com/downloads/) Update 3 or later.
```
Comment (scottaddie) [inline aspnetcore/mvc/razor-pages/index.md]: Update 3 --> 15.3

File: aspnetcore/mvc/razor-pages/index.md [hunk 5]
```diff
@@ -13,21 +13,19 @@ uid: mvc/razor-pages/index
...
-[Download or view sample code](https://github.com/Rick-Anderson/razor-page-intro).
-
 <a name="prerequisites"></a>
 
 ## ASP.NET Core 2.0 preview prerequisites
 
-ASP.NET Core 2.0 preview  is included in .NET Core 2.0 preview. [Download .NET Core 2.0 Preview 2](https://www.microsoft.com/net/core/preview) 
+ASP.NET Core 2.0 preview is included in .NET Core 2.0 preview. [Download .NET Core 2.0 Preview 2](https://www.microsoft.com/net/core/preview)
 
 If you’re using Visual Studio, [Visual Studio 2017 Preview version 15.3](https://www.visualstudio.com/vs/preview/) or later is required.
```
Comment (scottaddie) [inline aspnetcore/mvc/razor-pages/index.md]: Remove reference to preview

**Comment by Rick-Anderson** [comment]
@scottaddie  fixed - thanks.

**Pull request merged**